Module10.playground/Contents.swift

```diff
@@ -0,0 +1,62 @@
+import Foundation
+import PlaygroundSupport
+
+PlaygroundPage.current.needsIndefiniteExecution = true
```

Comment:
I see you used this because you were calling the function on a Task. Instead, you can remove this and just call the async function inline. The global context in Xcode Playgrounds can run async functions.